config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
+
+# Specify a serializer for the signed and encrypted cookie jars.
+# Valid options are :json, :marshal, and :hybrid.
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
TIL.

Can you add a link to that thing in the commit here with a message when to move to `:json`?

Reply:
We've expanded the commit message, and we'll open a separate PR with the change to `json` from `hybrid` as we'd like to discuss further when best this should be rolled out. 

From what we can see, it looks like the cookies for this app will expire after a day, so potentially apply the other PR after a day or so.

Reply:
👍